<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDeviceFileForm</class>
 <widget class="QWidget" name="CDeviceFileForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="prevToolButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="canary-img.qrc">
       <normaloff>:/img/ic_back</normaloff>
       <normalon>:/img/ic_back_white</normalon>:/img/ic_back</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="nextToolButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="canary-img.qrc">
       <normaloff>:/img/ic_forward</normaloff>
       <normalon>:/img/ic_forward_white</normalon>:/img/ic_forward</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="pathLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="refreshToolButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="canary-img.qrc">
       <normaloff>:/img/ic_open_folder</normaloff>
       <normalon>:/img/ic_open_folder_white</normalon>:/img/ic_open_folder</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="CDeviceFileTableWidget" name="fileTableWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDeviceFileTableWidget</class>
   <extends>QTableWidget</extends>
   <header>cdevicefileform.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="canary-img.qrc"/>
 </resources>
 <connections/>
</ui>
